package-lock.json

```diff
@@ -3123,40 +3123,13 @@
       }
     },
     "@wordpress/data-controls": {
-      "version": "1.3.4",
-      "resolved": "https://registry.npmjs.org/@wordpress/data-controls/-/data-controls-1.3.4.tgz",
-      "integrity": "sha512-Wt2sNMKAX/CKeeeQkdPtolTDcP1nPjvPnXiKLL116h89i4nJ3gBgS6PA31XEeDFuj/tq8HiYeMbzfijKhJyHsA==",
+      "version": "1.3.3",
+      "resolved": "https://registry.npmjs.org/@wordpress/data-controls/-/data-controls-1.3.3.tgz",
+      "integrity": "sha512-MA67FZi733IAfUGSqjt/J1yGyls4xgnl6Z+l57ihpEcB6QCxRhWS4Nl3nOuAkzgtPvPgbTQTns/c3dTDpda2uw==",
```

Comment:
🤔 huh!  I was wondering about this but I guess wp rolled back the patch versions (hence the changes in package-lock.json here for the `@wordpress/*` packages.)